frontend/src/scenes/billing/BillingLimitInput.tsx

```diff
@@ -13,87 +12,106 @@ import { billingProductLogic } from './billingProductLogic'
-                                </Tooltip>
-                            </>
-                        ) : (
-                            <>
-                                <Field name="input" noStyle>
-                                    {({ value, onChange, error }) => (
-                                        <Tooltip title={error}>
-                                            <div className="max-w-40">
-                                                <LemonInput
-                                                    ref={limitInputRef}
-                                                    type="number"
-                                                    fullWidth={false}
-                                                    status={error ? 'danger' : 'default'}
-                                                    value={value}
-                                                    onChange={onChange}
-                                                    prefix={<b>$</b>}
-                                                    disabled={billingLoading}
-                                                    min={0}
-                                                    step={10}
-                                                    suffix={<>/{billing?.billing_period?.interval}</>}
-                                                    size="small"
-                                                />
-                                            </div>
-                                        </Tooltip>
-                                    )}
-                                </Field>
+                    {!isEditingBillingLimit ? (
+                        <div className="flex items-center justify-center gap-1">
+                            {customLimitUsd ? (
+                                <>
+                                    <Tooltip title="Set a billing limit to control your recurring costs. Some features may stop working if your usage exceeds your limit.">
+                                        <span>
+                                            ${customLimitUsd} {billing?.billing_period?.interval}ly billing limit for{' '}
+                                            {product?.name}
+                                        </span>
+                                    </Tooltip>
+                                    <LemonButton
+                                        onClick={() => setIsEditingBillingLimit(true)}
+                                        status="danger"
+                                        size="small"
+                                    >
+                                        Edit limit
+                                    </LemonButton>
+                                </>
+                            ) : (
+                                <>
+                                    <span>You do not have a billing limit set for {product?.name}.</span>
```

Comment:
Think this copy is more straightforward. 

I don't particularly like how we repeat "set a billing limit" but think it's fine for now. 

<img width="915" alt="Screenshot 2024-06-05 at 2 22 25 PM" src="https://github.com/PostHog/posthog/assets/21014901/10aec55c-344c-4075-ae0f-60cf11feff34">

Reply:
Removed the product.name because it's already embedded in a product card.

Reply:
> Removed the product.name because it's already embedded in a product card.

Are you saying I should remove the product name? I added it just to be verbose since it's possible the product name it now in the viewport at the same time.

Reply:
I felt it was redundant. I also don't think our product blocks are tall enough that the product name would be out of the viewport unless one was on mobile? 

One way around that issue would be to move the billing limits block up, to be closer to the billing gauge. That might look really weird. 

Up to you whether to include the product name or not.